.concourse/pipeline.yaml

```diff
@@ -97,6 +97,7 @@ deploy_args: &deploy_args
   export SCF_CHART="$(readlink -f s3.kubecf-ci/*.tgz)"
   export SCF_OPERATOR=true
   export FORCE_DELETE=true
+  export HELM_VERSION="v3.1.1"
```

Comment:
@f0rmiga we tried to find some bazel command to extract this value from [here](https://github.com/cloudfoundry-incubator/kubecf/blob/master/def.bzl#L31) but we couldn't. Help is welcome.